ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -769,21 +769,20 @@ func (r *RayClusterReconciler) reconcilePods(ctx context.Context, instance *rayv
 		if worker.NumOfHosts <= 0 {
 			worker.NumOfHosts = 1
 		}
-		numExpectedPods := workerReplicas * worker.NumOfHosts
+		numExpectedPods := int(workerReplicas * worker.NumOfHosts)
```

Comment:
Converting `int32` to `int32` will not cause overflow.

Reply:
The code is doing the opposite, it converts to `int` instead of `int32`?

Reply:
Oh yeah, it is a typo in my comment.